src/routes/callbacks/index.ts

```diff
@@ -1,18 +1,21 @@
+import { deriveFromSecret, hmacSha256 } from '../../utils/secret'
 import { json, Router } from 'express'
 
 import { createLogger } from '../../factories/logger-factory'
 import { createSettings } from '../../factories/settings-factory'
 import { getRemoteAddress } from '../../utils/http'
+import { postLNbitsCallbackRequestHandler } from '../../handlers/request-handlers/post-lnbits-callback-request-handler'
 import { postZebedeeCallbackRequestHandler } from '../../handlers/request-handlers/post-zebedee-callback-request-handler'
 
 const debug = createLogger('routes-callbacks')
 
 const router = Router()
 router
-  .use((req, res, next) => {
+  .post('/zebedee', json(), (req, res) => {
```

Comment:
Can this be extracted into a method that both `/zebedee` and `/lnbits` uses with added checks that are unique for the different cases?  Both cases have some overlap, and that can be a method, and then all that remains is unique checks for each case. Let me know if that doesn't make sense and I can provide more context.